packages/babel-plugin-transform-typescript/src/enum.ts

```diff
@@ -131,109 +137,168 @@ const enumSelfReferenceVisitor = {
   ReferencedIdentifier,
 };
 
-export function translateEnumValues(
-  path: NodePath<t.TSEnumDeclaration>,
-  t: t,
-): Array<[name: string, value: t.Expression]> {
+export function translateEnumValues(path: NodePath<t.TSEnumDeclaration>, t: t) {
   const seen: PreviousEnumMembers = new Map();
   // Start at -1 so the first enum member is its increment, 0.
   let constValue: number | string | undefined = -1;
   let lastName: string;
 
-  return path.get("members").map(memberPath => {
-    const member = memberPath.node;
-    const name = t.isIdentifier(member.id) ? member.id.name : member.id.value;
-    const initializer = member.initializer;
-    let value: t.Expression;
-    if (initializer) {
-      constValue = evaluate(initializer, seen);
-      if (constValue !== undefined) {
-        seen.set(name, constValue);
-        if (typeof constValue === "number") {
-          value = t.numericLiteral(constValue);
+  return {
+    data: seen,
+    nodes: path.get("members").map(memberPath => {
```

Comment:
Since this is not an array of nodes, could we use another name? Like `enumValues`